src/base_output_updater.cpp

```diff
@@ -26,7 +26,7 @@ void compute_delta_z_output(std::vector<float> &mu_a, std::vector<float> &var_a,
     // We compute directely the inovation vector for output layer
     for (int col = start_chunk; col < end_chunk; col++) {
         tmp = jcb[col] / (var_a[col] + var_obs[col]);
-        if (isinf(tmp) || isnan(tmp)) {
+        if (isinf(tmp) || isnan(tmp) || isnan(obs[col])) {
```

Comment:
obs should not be nan, no? if for some reason it has to be,  we should handle it somewhere else

Reply:
I would like to keep it because in Hydro Quebec dataset, there are many time series with missing values, not at a single or several time steps, but for an extended period, e.g. years. In this case, we don't know which values should be filled to replace nan in the pre-processing step.

Reply:
ok.